packages/flutter/lib/src/widgets/service_extensions.dart

```diff
@@ -0,0 +1,86 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// Service extension that, when called, will output a string representation of
```

Comment:
nit (here and elsewhere):

```suggestion
/// Name of service extension that, when called, will output a string representation of
```